tasks/omnibus.py

```diff
@@ -69,11 +69,15 @@ def bundle_install_omnibus(ctx, gem_path=None, env=None, max_try=2):
 
         with gitlab_section("Bundle install omnibus", collapsed=True):
             for trial in range(max_try):
+                # The exception being raised when ommiting `warn=True` only includes
+                # the last 10 lines of stdout/stderr which might not be enough
+                # for should_retry_bundle_install to do its job
```

Comment:
Ah indeed I missed the fact that `UnexpectedExit` inherits from `Failure`

I'll look into leveraging this since it will allow us to simply propagate the exception which should be cleaner